make_prg/subcommands/update.py

```diff
@@ -0,0 +1,187 @@
+import multiprocessing
+import os
+from pathlib import Path
+from loguru import logger
+from make_prg.utils import io_utils, gfa
+from make_prg.update.denovo_variants import DenovoVariantsDB
+from make_prg.prg_builder import PrgBuilderZipDatabase, LeafNotFoundException
+from make_prg.utils.msa_aligner import MAFFT, MSAAligner
+from dataclasses import dataclass
+
+
+def register_parser(subparsers):
+    subparser_update_prg = subparsers.add_parser(
+        "update",
+        usage="make_prg update",
+        help="Update PRGs given new sequences.",
+    )
+
+    def check_if_is_update_file(argument: str):
+        argument = Path(argument)
+        is_zip_file = argument.suffix == ".zip"
+        is_update_DS = argument.suffix == ".update_DS"
+        if not is_zip_file and not is_update_DS:
+            subparser_update_prg.error(f"{argument} is not a update_DS nor a zip file.")
+        return argument
+    subparser_update_prg.add_argument(
+        "-u",
+        "--update_DS",
+        action="store",
+        type=lambda argument: check_if_is_update_file(argument),
+        required=True,
+        help=(
+            "Filepath to the update data structures (a *.update_DS.zip file created "
+            "by make_prg from_msa)"
+        ),
+
+    )
+    subparser_update_prg.add_argument(
+        "-d",
+        "--denovo_paths",
+        action="store",
+        type=str,
+        required=True,
+        help=(
+            "Filepath containing denovo sequences. Should point to a denovo_paths.txt file"
+        ),
+    )
+    subparser_update_prg.add_argument(
+        "-o",
+        "--output_prefix",
+        action="store",
+        type=str,
+        required=True,
+        help="Prefix for the output files",
+    )
+    subparser_update_prg.add_argument(
+        "-t",
+        "--threads",
+        action="store",
+        type=int,
+        default=1,
+        help="Number of threads (default 1)",
+    )
+    subparser_update_prg.add_argument(
+        "--mafft",
+        help="Path to MAFFT executable. By default, it is assumed to be on $PATH",
+        default="mafft",
+    )
+    subparser_update_prg.add_argument(
+        "--output_graphs",
```

Comment:
Add short option

Reply:
Done in https://github.com/iqbal-lab-org/make_prg/pull/35/commits/eee33ad6f266062c34a8f0918504cf0cb4a1793e